shared/src/business/useCases/serveCaseToIrs/generateDraftDocument.js

```diff
@@ -0,0 +1,35 @@
+const {
```

Comment:
Short as it might be, is it worth adding a test for this file?

Reply:
@TomElliottFlexion, I decided against a new test, because `serveCaseToIrsInteractor.test.js` already tests it. I can make that test fail by changing this function.